### PR TITLE
Emojis show as jumbo in main thread - Multi emoji

### PR DESCRIPTION
**Pull request by a-arias**
Signed-off-by: Abraham Arias <arias.abram@gmail.com>

#### Summary
Write an automated test using Cypress for "Emojis show as jumbo in main thread - Multi emoji, no text, including unicode and emoticon"

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17457
Fixes https://github.com/mattermost/mattermost-server/issues/11761

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

**Comment by a-arias** [comment]
@saturninoabril  This is ready to review.

**Comment by saturninoabril** [comment]
Hello @a-arias, sorry for the delay.  We're caught up by bulk of items for this coming release.  We'll review this next.

File: e2e/cypress/integration/messaging/emoji_size_spec.js [hunk 1]
```diff
@@ -84,4 +84,26 @@ describe('Messaging', () => {
         // # Making sure Emojis from last post message are 21px size
         checkEmojiSize('@newLineMessage', emojis, false);
     });
+
+    it('MM-17457 Unicode and emoticon', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/emoji_size_spec.js]:
```suggestion
    it('M17457 Emojis show as jumbo in main thread - Multi emoji, no text, including unicode and emoticon', () => {
```

File: e2e/cypress/integration/messaging/emoji_size_spec.js [hunk 2]
```diff
@@ -84,4 +84,26 @@ describe('Messaging', () => {
...
+
+        // # Post Emojis list
+        cy.postMessage(emojis.join(''));
+
+        // #Get last post message
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).as('lastMessage');
+
+            //# Expect unicode value from last message to have jumbo size
+            cy.get('@lastMessage').find('.emoticon--unicode').should('have.css', 'height', '32px').and('have.css', 'width', '32px');
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/emoji_size_spec.js]: ``.and('have.text', '🤟')`` --> Please include additional check.

**Comment by jasonblais** [comment]
Hey @a-arias, touching base to see if you have any questions about the feedback above from @saturninoabril? We'd be glad to help!

File: e2e/cypress/integration/messaging/emoji_size_spec.js (hunk 1 above)
Comment (jasonblais) [reply e2e/cypress/integration/messaging/emoji_size_spec.js]: Was `M17457` a typo? (and intended to be `MM-14757`)

Comment (a-arias) [reply e2e/cypress/integration/messaging/emoji_size_spec.js]: it was sorry, I just push a fix for this.

File: e2e/cypress/integration/messaging/emoji_size_spec.js (hunk 2 above)
Comment (a-arias) [reply e2e/cypress/integration/messaging/emoji_size_spec.js]: Done

**Comment by a-arias** [comment]
@jasonblais  @saturninoabril  I just fixed the minor changes, This is ready to review.

**Comment by jasonblais** [comment]
Thanks @a-arias! Re-requesting reviews

File: e2e/cypress/integration/messaging/emoji_size_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/messaging/emoji_size_spec.js]: @jasonblais That's correct.  It's the test key we're using to identify test case in our release spreadsheet. `M17457` means "M" as Messaging tab in release spreadsheet with "17457" as MM-17457 Jira ticket.  We documented this at - https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs

**Pull request merged**